spyderlib/config.py

```diff
@@ -510,6 +510,17 @@ def is_ubuntu():
               'editor/toggle comment': "Ctrl+1",
               'editor/blockcomment': "Ctrl+4",
               'editor/unblockcomment': "Ctrl+5",
+              'editor/start of line': "Meta+A",
+              'editor/end of line': "Meta+E",
+              'editor/previous line': "Meta+P",
+              'editor/next line': "Meta+N",
+              'editor/previous char': "Alt+B",
+              'editor/next char': "Alt+F",
+              'editor/previous word': "Meta+B",
+              'editor/next word': "Meta+F",
+              'editor/Kill to Line End': "Meta+K",

```

Comment:
What does `Kill` mean? Could you find a better word? ;-)

Also, I'd like (when possible) to use here the same shortcuts as the IPython console (just for consistency and muscle memory :-)

You can find them in

    Help > IPython doc > Console help